### PR TITLE
[UN-632] Read more / related resources styling

### PR DESCRIPTION
**Pull request by janehughes**
Ticket URL: https://national-archives.atlassian.net/jira/software/c/projects/UN/boards/75?modal=detail&selectedIssue=UN-632

## About these changes

Updates styles for related resource cards to match new designs.

## How to check these changes

Review story page with related resources and see that the new designs have been added.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [ ] Waited for all CI jobs to pass before requesting a review.
- [ ] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: sass/includes/_variables.scss [hunk 1]
```diff
@@ -33,6 +33,7 @@ $color__blue_5: #8ec0ec;
 $color__purple: #4c2c92;
 $color__red: #a72916;
 
+$color__light-grey: #fafafa;
```
Comment (nicklee) [inline sass/includes/_variables.scss:36]: This has been updated to #D9D9D6 to match the new brand book (https://www.figma.com/file/uo8YY98lLws8E1HEfpdn5X/TNA-Etna---Design-system?type=design&node-id=8%3A38&t=I6lb9pSno1ZGEVLp-1)

File: sass/includes/_related-resources.scss [hunk 2]
```diff
@@ -18,38 +18,53 @@
...
+    font-size: 1rem;
+
+    p {
+        margin-bottom: 0;
+    }
   }
 
   &__link, &__link-text {
-    color: $color__white !important;
+    color: $color__black !important;
```
Comment (nicklee) [inline sass/includes/_related-resources.scss:68]: shame we need to use !important, but guessing there's no other easy way around this?

File: sass/includes/_related-resources.scss [hunk 3]
```diff
@@ -18,38 +18,53 @@
...
+    //   outline-offset: .25rem;
+    // }
   }
 
   &__link-heading {
     margin-top: 0;
-    font-size: 1.625rem;
+    font-family: $font__body;
+    font-weight: $font-weight-bold;
+    font-size: 1.125rem;
```
Comment (nicklee) [inline sass/includes/_related-resources.scss:51]: Can we use the font-size mixin for font sizes so these tie in with the sizes defined in the styleguide? If it's too much of a faff I can go through and refactor fonts later on https://github.com/nationalarchives/ds-wagtail/blob/686a4a07acd1cecfc7a01ac63ddd548712c77db0/sass/includes/_mixins.scss#L35

**Review by nicklee — APPROVED** [review]
All looks good, a few minor optional comments, mainly around font sizes and whether we can use the mixin, and the updated colours since we integrated the new brand book

File: sass/includes/_variables.scss (hunk 1 above)
Comment (nicklee) [reply sass/includes/_variables.scss:36]: Just checking the component design, you were right to use #fafafa but this maps to grey-100 in the design system, light-grey is a brand colour defined at #d9d9d6, which may confuse things… I guess we could do with updating the codebase to match Figma or vice-versa

Comment (janehughes) [reply sass/includes/_variables.scss:36]: ah okay bad naming choice on my part! shall I update this to grey-100 instead in the variables file and use that?

File: sass/includes/_related-resources.scss (hunk 3 above)
Comment (janehughes) [reply sass/includes/_related-resources.scss:51]: I'm going to keep this consistent with the other PR so will leave it for now. Can come back and add in the font mixin once we've got the sizing updated across the board 👍  just to save any inconsistencies or discrepancies between design/FE.

File: sass/includes/_related-resources.scss (hunk 2 above)
Comment (janehughes) [reply sass/includes/_related-resources.scss]: I've removed this now :) wasn't needed!

**Pull request merged**